schemas/iso19115-3.2018/src/main/plugin/iso19115-3.2018/convert/protocol-mapping.xsl

```diff
@@ -0,0 +1,55 @@
+<xsl:stylesheet version="2.0"
+                xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+                xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                xmlns:dct="http://purl.org/dc/terms/"
+                xmlns:dcat="http://www.w3.org/ns/dcat#"
+                xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+                exclude-result-prefixes="#all">
+
+    <xsl:output method="xml" indent="yes"/>
+    <xsl:strip-space elements="*"/>
+
+    <xsl:variable name="format-mimetype-mapping">
+        <entry>
+            <format>csv</format>
+            <mimetype>text/csv</mimetype>
+            <protocol>WWW:DOWNLOAD</protocol>
+        </entry>
+        <entry>
+            <format>geojson</format>
+            <mimetype>application/vnd.geo+json</mimetype>
+            <protocol>WWW:DOWNLOAD</protocol>
+        </entry>
+        <entry>
+            <format>kml</format>
+            <mimetype>application/vnd.google-earth.kml+xml</mimetype>
+            <protocol>WWW:DOWNLOAD</protocol>
+        </entry>
+        <entry>
+            <format>zip</format>
+            <mimetype>application/zip</mimetype>
+            <protocol>WWW:DOWNLOAD</protocol>
+        </entry>
+        <entry>
+            <format>shapefile</format>
+            <mimetype>application/zip</mimetype>
```

Comment:
Actually I would use ` x-gis/x-shapefile` here, see: https://en.wikipedia.org/wiki/Shapefile

The actual file will be a `.zip` and not `.shp`, but without this we'll never be able to exploit the file without unzipping it. So it's mure future-proof that way I think.